apps/Wallet/src/app/api/safe/users/is-user/route.ts

```diff
@@ -0,0 +1,14 @@
+import { AddressTransactionBody } from "@/types";
+import { Rbac__factory } from "../../../../../types/typechain/types/config/abis";
+import { getEtherscanSigner } from "../../util/utils";
+
+export async function GET(req: Request, res: Response): Promise<Response> {
+    const body = (await req.json()) as AddressTransactionBody;
+    const signer = getEtherscanSigner(process.env.OWNER_1_PRIVATE_KEY_GOERLI!);
+    const rbacModuleAddress = process.env.RBAC_MODULE_ADDRESS!;
+    const rbac = Rbac__factory.connect(rbacModuleAddress, signer);
+    const safe = process.env.SAFE_ADDRESS!;
+    const isUser = await rbac.isDelegate(safe, body.address)
```

Comment:
This user does not require a private key so lets just pass in the provider instead of signer

Reply:
fixed

Reply:
same here, can just use getEthAdapter and use the `.getProvider` function